src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -261,43 +277,6 @@ export default class MutationMapperStore extends DefaultMutationMapperStore {
         return lazyMobXTableSort(this.mergedAlignmentData, sortMetric, false);
     }
 
-    @computed get transcriptsByTranscriptId(): {
```

Comment:
This seems duplicate of https://github.com/cBioPortal/cbioportal-frontend/blob/2f549e804f1dfa579bbd8fcb6ba0f7ddbd340007/packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts#L329